pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -178,61 +178,33 @@ func (r *ReconcileChaosEngine) Reconcile(request reconcile.Request) (reconcile.R
 // getChaosRunnerENV return the env required for chaos-runner
 func getChaosRunnerENV(cr *litmuschaosv1alpha1.ChaosEngine, aExList []string, ClientUUID string) []corev1.EnvVar {
 
-	var appNS string
-
-	if cr.Spec.Appinfo.Appns != "" {
-		appNS = cr.Spec.Appinfo.Appns
-	} else {
+	appNS := cr.Spec.Appinfo.Appns
```

Comment:
I think you should invert this.
```suggestion
	appNS := cr.Namespace // as this will always exist.
	if cr.Spec.Appinfo.Appns != "" {
		appNS = cr.Spec.Appinfo.Appns
	}
```

Reply:
This is fine. if `cr.Spec.Appinfo.Appns` is defined then it will use it otherwise it will use cr.Namespace

Reply:
Both are fine -- one branch in both cases

Reply:
I just hoped we could eliminate `else` statement here. No problem tho.

Reply:
I haven't used else here

Reply:
Sounds good then.